tensorflow/compiler/xla/service/cus_related_functions.cc

```diff
@@ -81,7 +29,7 @@ StatusOr<llvm::Value*> EmitF32ToCus(llvm::Value* f32_value,
 
 llvm::Value* EmitCusNeg(llvm::Value* cus_value, llvm::IRBuilder<>* b) {
```

Comment:
You probably need to add another macro for registering unary operators too. May be not urgent right now.

Reply:
Yes, I was thinking to add that when needed